index.html

```diff
@@ -0,0 +1,299 @@
+        </div>
+      </div>
+      <div class="container-projects-grid">
+        <div class="see">
+          <button class="change">See project</button>
+        </div>
+
+        <div class="data-dashbord">
+          <h4>Data Dashbord <br />Healthcare</h4>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </p>
+          <div class="button">
+            <button>HTML</button>
+            <button>Bootstrap</button>
+            <button>Ruby</button>
+          </div>
+          <button class="see">See project</button>
+        </div>
+        <div class="data-dashbord">
+          <h4>Website portfolio</h4>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </p>
+          <div class="button">
+            <button class="html">HTML</button>
+            <button>Bootstrap</button>
+            <button>Ruby</button>
+          </div>
+          <button class="see">See project</button>
+        </div>
+        <div class="data-dashbord">
+          <h4>Profesional Art <br />Printing Data More</h4>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </p>
+          <div class="button">
+            <button class="html">HTML</button>
+            <button>Bootstrap</button>
+            <button>Ruby</button>
+          </div>
+          <button class="see">See project</button>
+        </div>
+        <div class="data-dashbord">
+          <h4>Data Dashbord <br />Healthcare</h4>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </p>
+          <div class="button">
+            <button class="html">HTML</button>
+            <button>Bootstrap</button>
+            <button>Ruby</button>
+          </div>
+          <button class="see" >See project</button>
+        </div>
+        <div class="data-dashbord">
+          <h4>Profesional Art <br />Printing Data More</h4>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </p>
+          <div class="button">
+            <button class="html">HTML</button>
+            <button>Bootstrap</button>
+            <button>Ruby</button>
+          </div>
+          <button  class="see">See project</button>
+        </div>
+      </div>
+    </section>
+    <!-- Project section Ends -->
+
+    <!-- About Me Section Starts -->
+    <section class="about-me" id="about-me">
+      <div class="container">
+        <div class="about-me">
+          <p>About me</p>
+          <p>
```

Comment:
Just like in my previous comment about adding a heading to give the flow of  the content of your page a form of hierarchy, I think that you should change this p tag (line 212) to an h2 tag or lower to improve upon the semantics of your page